vignettes/cookbook_running_the_analysis.Rmd

```diff
@@ -14,6 +14,246 @@ knitr::opts_chunk$set(
+
+In the example below, we see two classification codes coming from the SIC classification standard:
+
+``` {r}
+r2dii.data::nace_classification %>% 
+  filter(code %in% c("D35.11", "D35.14"))
+```
+
+Notice that the code D35.11 corresponds to power generation. This is an identical match to PACTA's `power` sector, and thus the `borderline` flag is set to `FALSE`. In contrast, code D35.14 corresponds to the distribution of electricity. In a perfect world, we would set this code to `not in scope`, however there is still a chance that these companies produce electricity. For this reason, we have mapped it to `power` with `borderline = TRUE`.
+
+In practice, if a company has a `borderline` of `TRUE` and _is_ matched, then consider the company in scope. If it has a `borderline` of `TRUE` and _isn't_ matched, then consider it out of scope.
+
+## Calculate targets
+
+You can calculate scenario targets using two different approaches: Market Share Approach, or Sectoral Decarbonization Approach.
+
+### Market Share Approach
+
+The [Market Share Approach](https://rmi-pacta.github.io/r2dii.analysis/articles/target-market-share.html) is used to calculate scenario targets for the `production` of a technology in a sector. For example, we can use this approach to set targets for the production of electric vehicles in the automotive sector. This approach is recommended for sectors where a granular technology scenario roadmap exists.
+
+Targets can be set at the portfolio level:
+
+```{r}
+# Use these datasets to practice but eventually you should use your own data.
+scenario <- r2dii.data::scenario_demo_2020
+regions <- r2dii.data::region_isos_demo
+
+market_share_targets_portfolio <-
+  target_market_share(
+    data = prioritized_matches,
+    abcd = abcd,
+    scenario = scenario,
+    region_isos = regions
+  )
+
+market_share_targets_portfolio
+```
+
+Or at the company level: 
+
+```{r market-share-targets-company}
+market_share_targets_company <-
+  target_market_share(
+    data = prioritized_matches,
+    abcd = abcd,
+    scenario = scenario,
+    region_isos = regions,
+    by_company = TRUE # Output results at company-level
+  )
+
+market_share_targets_company
+```
+
+### Sectoral Decarbonization Approach
+
+The [Sectoral Decarbonization Approach](https://rmi-pacta.github.io/r2dii.analysis/articles/target-sda.html) is used to calculate scenario targets for the `emission_factor` of a sector. For example, you can use this approach to set targets for the average emission factor of the cement sector. This approach is recommended for sectors lacking technology roadmaps.
+
+```{r sda-targets}
+# Use this dataset to practice but eventually you should use your own data.
+co2 <- r2dii.data::co2_intensity_scenario_demo
+
+sda_targets <-
+  target_sda(
+    data = prioritized_matches,
+    abcd = abcd,
+    co2_intensity_scenario = co2,
+    region_isos = regions
+  ) %>% 
+  filter(sector == "cement", year >= 2020)
+
+sda_targets
```

Comment:
Please add links at the bottom to navigate previous and next chapters:
PREVIOUS CHAPTER: [Running the Analysis](link)

NEXT CHAPTER: [Interpretation of Results](link)

Reply:
done